### ISSUE TITLE
Leave Insert mode

### ISSUE THREAD

**Issue by julienXX** [skwp/vim-ruby-conque#7]
Hi,
How can I leave insert mode after running my specs? I tried let g:ConqueTerm_InsertOnEnter = 0 but no luck.

Cheers,
Julien.

**Comment by skwp** [comment]
Yeah I found that it for some reason enters insert..You could always hit `<esc>` :)
I am working on ideas such as sending `<esc>` to it automatically.

**Comment by skwp** [comment]
Are you using the YADR project (my dotfiles project)? I noticed I had InsertOnEnter=1 so it might have been overriding your settings

**Comment by julienXX** [comment]
No I'm using my own dot files. This must be something with Conque not taking this option into account.

**Comment by rspeicher** [comment]
Seeing this as well, even with `let g:ConqueTerm_InsertOnEnter = 0` set.

**Comment by comeara** [comment]
Has anyone come up with a workaround for this? I installed the plugin and it's exactly what I've been looking for. Insert mode is an annoyance though.

I tried g:ConqueTerm_InsertOnEnter = 0 without success.

**Comment by cvincent** [comment]
Indeed! Any workaround for this yet?

**Comment by comeara** [comment]
No, no workaround.

The annoyance was great enough that I gave up trying to use the plugin.

**Comment by skwp** [comment]
I'm annoyed also :) but then all you really gotta do is hit escape.
